.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      # TODO: Should be scoped to tag pushes eventually
+
+jobs:
+  docker_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ngrok/ngrok-ingress-controller:latest
+    
+  helm_release:
+    needs: docker_release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
so this will show the commit as the user who submitted the PR, like you see [here](https://github.com/ngrok/ngrok-ingress-controller/commits/gh-pages).  I didn't /directly/ make these commits, the GH Action did, but the delta(s) are attributed to me because I submitted the PR.   Here's the doc for [Github Actions Environment Variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)